File: MyClub/MyClub.Model/Responses/DashboardMembershipPerMonthResponse.cs
using System;

namespace MyClub.Model.Responses
{
    public class DashboardMembershipPerMonthResponse
    {
        public string Month { get; set; }
        public int Year { get; set; }
        public string MonthName { get; set; }
        public int Count { get; set; }
    }
} 

File: MyClub/UI/myclub_desktop/lib/models/dashboard/dashboard_membership_per_month_response.dart
class DashboardMembershipPerMonthResponse {
  final String month;
  final int year;
  final String monthName;
  final int count;

  DashboardMembershipPerMonthResponse({
    required this.month,
    required this.year,
    required this.monthName,
    required this.count,
  });

  factory DashboardMembershipPerMonthResponse.fromJson(Map<String, dynamic> json) {
    return DashboardMembershipPerMonthResponse(
      month: json['month'],
      year: json['year'],
      monthName: json['monthName'],
      count: json['count'],
    );
  }
}
